documentation/Ingress-controller-upgrade.md

```diff
@@ -0,0 +1,36 @@
+# Ingress Controller Upgrade
+
+**Pre-requisits**
+- AKS Cluster with the ingress controller you wish to upgrade
+- An app deployed to the cluster to monitor i.e. apply, register etc
+- Set you AKS context for the cluster `az aks get-credentials ...`
+
+**Monitoring the upgrade**
+- Start the monitoring in a separate window using curl. This will monitor any app downtime please open a new window or terminal session for this and suite the url your current app deployment
```

Comment:
Repetition: "in a separate window" and "please open a new window or terminal session for this"